server/src/main/java/de/hellfish/ngrok/server/ClientConnectionServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package de.hellfish.ngrok.server;
+
+import org.springframework.stereotype.Service;
+
+import java.net.Socket;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Service
+public class ClientConnectionServiceImpl implements ClientConnectionService {
+    private final ConcurrentHashMap<String, Socket> clientConnections = new ConcurrentHashMap<>();
```

Comment:
empty line between class declaration and a field

Reply:
To make realization and definition separate. Code looks better, easier to support. App is more flexible and finally support better polymorphism

Reply:
Solved